eng/scripts/TypeSpec-Requirement.ps1

```diff
@@ -20,9 +20,27 @@ function Get-Suppression {
 
   # -NoEnumerate to prevent single-element arrays from being collapsed to a single object
   # -AsHashtable is closer to raw JSON than PSCustomObject
-  $suppressions = npx get-suppressions TypeSpecRequirement $fileInSpecFolder | ConvertFrom-Json -NoEnumerate -AsHashtable
+  $suppressions = npm exec --no -- get-suppressions TypeSpecRequirement $fileInSpecFolder | ConvertFrom-Json -NoEnumerate -AsHashtable
 
-  return $suppressions ? $suppressions[0] : $null;
+  if ($LASTEXITCODE -ne 0) {
```

Comment:
Unrelated to crux of PR.  But if `get-suppressions` fails, either because user didn't run `npm ci` and tool isn't compiled, or because say `suppressions.yaml` is malformed and `get-suppressions` throws, we should fail fast.